.github/renovate.json5

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": [
+    "config:base",
+    // Disable the creation of this issue that renovate updates with the pending issue we follow.
+    ":disableDependencyDashboard"
+  ],
+  "enabledManagers": [
+    // Managers for helm and helm-values. Go dependencies are managed by Dependabot.
+    "helm-values",
+    "helmv3"
+  ],
```

Comment:
This misses bumping the version of Prometheus. You can inspire yourself with this: https://github.com/newrelic/nri-kubernetes/pull/541/files#diff-20ab1190d08a53a3012bc191ed56205c8f0ffb8fa1715e9d36ecfd1d2ea8a790R14-R22

Think that the regex needs that lines are one next to the other. This is why, in that PR, also changed values order: https://github.com/newrelic/nri-kubernetes/pull/541/files#diff-b1c831b81fe52267808d0d6996f50be42192cf1afbf1cd8502d6313db9ed8f67R23-R27

I am marking this as a NIT. We can open an issue and leave it until we choose which version of Prometheus Agent are we going to support because it could be hard to renovatebot to match LTS versions.

Reply:
> it could be hard to renovatebot to match LTS versions.
I did not add ti for that reason.
Currently dependabot will trigger the dependency update of Prometheus just for the LTS version and that PR will fail because the chart need also to be in that version. So either way if we could add this to renovate the PR will also fail ,since both needs to be sync at the same time.